cluster/pre-existing/util.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A library of helper functions for a pre-existing Kubernetes cluster
```

Comment:
Maybe make this more indicative of the assumption that "the kubemark master is pre-existing"? Pre-existing kubernetes cluster might be confusing as it could also mean the base cluster.